README.md

```diff
@@ -35,7 +35,7 @@ See also the following resources:
 If you are using luarocks, execute the following command to install the plugin:
 
 ```bash
-luarocks install kong-oauth-proxy
+luarocks install kong-oauth-proxy 1.3.0
```

Comment:
Should this say `1.3.0-1`? That is the version set in rockspec file.

Reply:
The -1 is just a LUAROCKS technology specific thing. Maybe both will work, though I'd like the reader instructions to use the most meaninful semantic version.